app/assets/stylesheets/govuk_publishing_components/components/_chat-entry.scss

```diff
@@ -0,0 +1,27 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.gem-c-chat-entry {
+  display: flex;
+  align-items: flex-start;
+}
+
+.gem-c-chat-entry--border-top {
+  border-top: 2px solid govuk-colour("blue");
+  padding-top: govuk-spacing(5);
+}
+
+.gem-c-chat-entry--border-bottom {
+  border-bottom: 2px solid govuk-colour("blue");
+  padding-bottom: govuk-spacing(5);
+}
+
+.gem-c-chat-entry__image {
+  flex-shrink: 0;
+  margin-right: govuk-spacing(3);
+}
+
+.gem-c-chat-entry__heading {
+  @include govuk-typography-common;
+  @include govuk-font-size(24);
```

Comment:
Nitpick: I think all includes like this need to now follow regular lines of CSS, rather than precede them, owing to a problem with a dependency that @kevindew fixed.